spec/Section 6 -- Execution.md

```diff
@@ -649,10 +806,26 @@ CompleteValue(fieldType, fields, result, variableValues):
   * If {result} is {null} (or another internal value similar to {null} such as
     {undefined} or {NaN}), return {null}.
   * If {fieldType} is a List type:
+    * If {result} is an iterator:
+      * Let {field} be the first entry in {fields}.
+      * Let {innerType} be the inner type of {fieldType}.
+      * Let {streamDirective} be the `@stream` directived provided on {field}.
+      * Let {initialCount} be the value or variable provided to {streamDirective}'s {initialCount} argument.
+      * Let {label} be the value or variable provided to {streamDirective}'s {label} argument.
+      * Let {resolvedItems} be an empty list
+      * For each {members} in {result}:
+        * Append all items from {members} to {resolvedItems}.
+        * If the length of {resolvedItems} is greater or equal to {initialCount}:
+          * Let {initialItems} be the sublist from the beginning to the {initialCount}-th item.
+          * Let {remainingItems} be the sublist from {initialCount}-th + 1 item to the end.
```

Comment:
Might be clearer to go with something that doesn't depend on indexes (which could be 0- or 1- indexed), such as:

```suggestion
          * Let {initialItems} be the sublist of the first {initialCount} items from {resolvedItems}.
          * Let {remainingItems} be the sublist of the items in {resolvedItems} after the first {initialCount} items.
```